### PR TITLE
Week 12: Learn Elm & Build Something Useful for F&C

### PR DESCRIPTION
**Pull request by nelsonic**
+ adds some background info for the week.
+ learning outcomes
+ instructions for what to do in the "How?" section of Readme.
+ "Final Project" will focus on building something _useful_ for F&C!

### CODE REVIEW

**Comment by andrewMacmurray** [comment]
@nelsonic, this is looking good

**Comment by sofiapoh** [comment]
Nice! I don't know how the frontend masters Elm tutorials are because I haven't seen them but when I started learning I did the egghead ones and those were great too  👍

**Comment by nelsonic** [comment]
I don't know people have seen: https://github.com/rtfeldman/elm-workshop/issues/8 💭

**Comment by nelsonic** [comment]
@iteles / @bradreeder / @des-des would _one_ of you mind reading through 
`coursebook/week-12/project.md` in this PR to confirm if the instructions are clear. 🔍 
please/thanks! ❤️

File: coursebook/week-12/project.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+it's time to _use_ those super-powers for _good_.
+
+> great responsibility
+
+## _Why_?
+
+### Founders & Coders and all future technology learners need(s) _your_ help!
+
+`while` _we_ have come a _long_ way in the past few years,
+there are still _many_ areas of F&C that _can_ be ***improvededed***.
```
Comment (iteles) [inline coursebook/week-12/project.md]: Deliberate typo, I can actually hear you say this...

**Comment by iteles** [comment]
There's so much great content here! It could do with a little re-ordering for cleaner flow in some places, but in the interest of having something available for FAC9 *tomorrow* should they wish to take this on instead of curriculum planning (pushing that to next week which I think @sofer mentioned was a possibility) *and* so that there's a starting point to build on, I'm merging this in.

Lots of iterations to come I'm sure but at least a first proposal for weeks 11 & 12 (redux and functional programming + Elm and a hack day) is in 👍

**Comment by des-des** [comment]
Just to be clear, does the merging of this pull request indicate that FAC has decided to add elm to the core curriculum?

**Comment by nelsonic** [comment]
@des-des great question! 
The _poll_ suggested that people _wanted_ to learn it:
anon-elm-lang-twitter-poll
But I'm not "_fussed_" if anyone decides to _remove_ it. 
or a mentor/cohort choses to learn something `else` instead.

Ultimately we should let people in the "_self-organising_" groups of learners 
decide what they want to learn. 
but as far as a _suggestion_ for learning something _incredibly_ useful/practical, 
`elm-lang` would be my choice over virtually everything `else`.

**Comment by iteles** [comment]
It seems like there's still debate in the issues about weeks 11 and 12. As this is the least defined portion, it's a fully formed 2 weeks, which can be iterated on as soon as someone else wants to take them on :+1:

**Comment by des-des** [comment]
@iteles I disagree. Before we iterate, we need to make a formal decision on if we want elm in the curriculum.

See the mission statement
> to open access to education by building democratically organised learning communities where people study creative technology.

**Comment by sofer** [comment]
There appear to be several parallel discussions going on about what the latter part of the programme could look like for FAC10 and NAZFAC1 (sorry! working title). so far as I am aware, we only have a consensus on Weeks 1-8. It looks like there is now a curriculum committee being convened by @bradreeder which I hope can make a recommendation to our next business meeting after consulting as widely as possible. We need to get our skates on! 

Whether this material is used for Weeks 11 & 12 or some other weeks in future, I love that we are attempting something new and I love that it's Elm.

**Pull request merged**